src/xdocs/google_style.xml

```diff
@@ -1485,6 +1501,27 @@
                     test</a>
                 </td>
               </tr>
+              <tr>
+                <td><a name="4.8.5.5"/>
+                  <a href="#4.8.5.5">
+                    <span class="wrapper inline">
+                      <img src="images/anchor.png" alt=""/>
+                    </span>
+                  </a>
+                </td>
+                <td>
+                  <a href="styleguides/google-java-style-20220203/javaguide.html#s4.8.5.5-local-parameter-annotation-style  ">
+                    4.8.5.5 Parameter and local variable annotations
+                  </a>
+                </td>
+                <td>
+                  -- For type-use annotations see
```

Comment:
Please ad "br" tag after `--` to make it clear it general symbol as in Other chapters 
Text should be on next line.

Reply:
done